<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>412</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>412</height>
   </size>
  </property>
  <property name="font">
   <font>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{
	background-color : rgb(204, 214, 217);
	border : outset 1px grey;
}
#cancelSearch:pressed{
background-color :rgb(255, 255, 255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lab_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>179</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Symbol</family>
       <pointsize>20</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Address Book</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>2</y>
       <width>146</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color : grey;</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QToolButton" name="cancelSearch">
     <property name="geometry">
      <rect>
       <x>552</x>
       <y>7</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none;</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="search_gp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>181</width>
      <height>331</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <widget class="QListWidget" name="lst_search">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>24</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Details">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter Details :</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>321</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="otherDetails">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lab_occupation">
        <property name="text">
         <string>Occupation :   </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_occupation">
        <property name="styleSheet">
         <string notr="true">background - color : blue;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lab_mobile">
        <property name="text">
         <string>Mobile : </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txt_mobile"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lab_work">
        <property name="text">
         <string>Work : </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txt_work"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lab_addr">
        <property name="text">
         <string>Address : </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txt_addr"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lab_email">
        <property name="text">
         <string>Email :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txt_email"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lab_comments">
        <property name="text">
         <string>Comments : </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txt_comments"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>251</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="name_age">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lab_name">
        <property name="text">
         <string>Name : </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txt_name"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lab_age">
        <property name="text">
         <string>Age : </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txt_age"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_image">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/rsc/face.png</normaloff>:/rsc/face.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>178</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_new">
        <property name="text">
         <string>New</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>search</tabstop>
  <tabstop>txt_name</tabstop>
  <tabstop>txt_age</tabstop>
  <tabstop>txt_occupation</tabstop>
  <tabstop>txt_mobile</tabstop>
  <tabstop>txt_work</tabstop>
  <tabstop>txt_addr</tabstop>
  <tabstop>txt_email</tabstop>
  <tabstop>txt_comments</tabstop>
  <tabstop>btn_save</tabstop>
  <tabstop>btn_image</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_save</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>searchText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelSearch</sender>
   <signal>clicked()</signal>
   <receiver>search</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_new</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>newEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lst_search</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>contactClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_image</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveEntry()</slot>
  <slot>changeImage()</slot>
  <slot>searchText(QString)</slot>
  <slot>contactClicked(QListWidgetItem*)</slot>
  <slot>newEntry()</slot>
  <slot>deleteEntry()</slot>
 </slots>
</ui>
